src/index.js

```diff
@@ -64,14 +71,54 @@ export function filterCards(cards, options) {
 	return cards.filter(card => !card.back.some(token => token.content.trim().includes('<!-- md2anki ignore-card -->'.trim())));
 }
 
-export function deckFromCards(cards, options) {
+export function deckFromCards(cards, images, options) {
 	// create new deck
 	const apkg = new AnkiDeck(options.deckName, { css: '' });
 	console.log(`deck initialized!`);
+	// add media files to deck
+	images.forEach(image => {
+		apkg.addMedia(image.replace('/', '#'), fs.readFileSync(image));
+	});
 	// add cards to deck (convert tokens to html)
-	cards.forEach((card, i) => {
-		apkg.addCard(md.renderer.render(card.front, md.options, {}), md.renderer.render(card.back, md.options, {}));
+	cards.forEach(card => {
+		apkg.addCard(md.renderer.render(card.front, md.options, {}), fixImagePaths(images, md.renderer.render(card.back, md.options, {})));
 	});
 	console.log(`added ${cards.length} cards to the deck!`);
+	console.log(`added ${images.length} images to the deck!`);
 	return apkg;
+}
+
+// make sure all images references point to the correct image file (/ replaced with #)
+export function fixImagePaths(images, html) {
+	images.forEach(image => {
+		html = html.replace(image, image.replace('/', '#'));
+	});
+
+	return html;
+}
+
+export function imagesFromTokens(tokens, inputPath) {
+	let images = [];
+	tokens.forEach((token) => {
+		if (token.type === 'image') {
+			// find images in tokens and save them to a list
+			let filePath = path.join(path.dirname(inputPath), token.attrGet('src'));
+			// replace \ with / (in case we're on a windows system)
+			filePath = filePath.split(path.sep).join(path.posix.sep);
+			images.push(filePath);
```

Comment:
Statt `fixImagePaths()` zu nutzen, würde ich vielleicht direkt hier per `token.attrSet('src', escapedPath)` die modifizierten Pfade setzen. Hat einfach damit zu tun, dass ich in den anderen PRs die Cards in eine einfache Klasse gepackt habe, und `renderToHTML` auch dort implementiert habe. Das mit der Funktion wäre also nicht in der Form umsetzbar.

Sowas tut mir echt leid, die Codebase wandelt sich hier am Anfang noch ziemlich stark, wenn du willst, kann ich das auch selbst refactoren?

Reply:
Ah jo, das wollte ich eigentlich auch so machen, wusste aber nicht, wie ich die tokens dann tatsächlich hätte speichern können.

Vielleicht können wir ja auch erstmal die anderen PRs mergen und dann schau ich nochmal in Ruhe. 
Weiß aber noch nicht, ob ich unter der Woche direkt schon dazu kommen werd.
Ist aber nicht schlimm, wenn ich das nochmal ändern muss.